pkg/dispatchcli/cmd/get_api.go

```diff
@@ -85,9 +85,25 @@ func formatAPIOutput(out io.Writer, list bool, apis []v1.API) error {
 		table.Append([]string{
 			*a.Name, *a.Function,
 			strings.Join(a.Protocols, "\n"), strings.Join(a.Methods, "\n"), strings.Join(a.Hosts, "\n"),
-			strings.Join(a.Uris, "\n"), a.Authentication, string(a.Status), fmt.Sprintf("%t", a.Enabled),
+			strings.Join(addHostName(a.Uris, a.Protocols), "\n"), a.Authentication, string(a.Status), fmt.Sprintf("%t", a.Enabled),
 		})
 	}
 	table.Render()
 	return nil
 }
+
+func addHostName(uris []string, protocals []string) []string {
+	var newUris []string
+	for _, uri := range uris {
+		for _, protocal := range protocals {
+			if protocal == "http" {
+				newUris = append(newUris, "http://"+dispatchConfig.Host+":8081"+uri)
```

Comment:
This really should be able to be simplified to
```
for _, protocol := range protocols {
    newUris = append(newUris, fmt.Sprintf("%s://%s:%s%s", protocol, dispatchConfig.Host, portMap[protocol], uri))
}
```
where portMap is some local or package variable that contains a map of the protocol to the API gateway port for that protocol. We really do need a way of adding the correct port information here. Hard coding 8081 or 8444 won't work.

Reply:
I find api ports in dispatchConfig, just update to: if the port is 0 then use default port